zib/StructureDefinitions/zib-TimeInterval.xml

```diff
@@ -0,0 +1,113 @@
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-timeinterval-v1.0-2024EN" />
+    <uri value="https://zibs.nl/wiki/TimeInterval-v1.3(2024EN)" />
+    <name value="zib TimeInterval-v1.3(2024EN)" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="true" />
+  <type value="Period" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Period" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Period">
+      <path value="Period" />
+      <short value="TimeInterval" />
+      <alias value="TijdsInterval" />
+      <mapping>
+        <identity value="zib-timeinterval-v1.0-2024EN" />
+        <map value="NL-CM:20.3.1" />
+        <comment value="TimeInterval" />
+      </mapping>
+    </element>
+    <element id="Period.extension:duration">
+      <path value="Period.extension" />
+      <sliceName value="duration" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-TimeInterval.Duration" />
+      </type>
+    </element>
+    <element id="Period.start">
+      <path value="Period.start" />
+      <short value="StartDateTime" />
+      <definition value="The start date and time of the interval." />
+      <alias value="StartDatumTijd" />
+      <mapping>
+        <identity value="zib-timeinterval-v1.0-2024EN" />
+        <map value="NL-CM:20.3.2" />
+        <comment value="StartDateTime" />
+      </mapping>
+    </element>
+    <element id="Period.start.extension:relativeStartTime">
+      <path value="Period.start.extension" />
+      <sliceName value="relativeStartTime" />
+      <definition value="Textual representation of a start time associated with an event for which the time is not yet fixed or known. For example &quot;Start the day after discharge from hospital&quot;." />
+      <type>
+        <code value="Extension" />
+        <profile value="https://example.org/fhir/StructureDefinition/ext-TimeInterval.RelativeStartTime" />
+      </type>
+      <isModifier value="false" />
+    </element>
+    <element id="Period.start.extension:relativeStartTime.value[x]">
+      <path value="Period.start.extension.value[x]" />
+      <alias value="RelatieveStartTijd" />
+      <mapping>
+        <identity value="zib-timeinterval-v1.0-2024EN" />
```

Comment:
Need to move mapping to extension - see discussion in our chat https://teams.microsoft.com/l/message/19:e2d139043d7947298963fb20b289091f@thread.v2/1732716410481?context=%7B%22contextType%22%3A%22chat%22%7D